packages/server/src/validators/index.ts

```diff
@@ -5,6 +5,10 @@ export const emailValidator = z.object({
     email: z.string().email(),
 });
 
+export const googleLoginSchema = z.object({
+    token: z.string().min(1),
```

Comment:
Is there a way to be more specific like what is the actual length of the string return by the react-google-login